src/structures/idt.rs

```diff
@@ -638,6 +654,27 @@ macro_rules! impl_set_handler_fn {
             pub fn set_handler_fn(&mut self, handler: $h) -> &mut EntryOptions {
                 self.set_handler_addr(handler as u64)
             }
+
+            const_fn! {
+                /// Set the handler function for the IDT entry and sets the present bit.
+                ///
+                /// # Safety
+                ///
+                /// For the code selector field, this function uses the code segment selector that
+                /// is passed in through the arguments, the user must take care to ensure that it is
+                /// valid.
+                ///
+                /// The function returns a mutable reference to the entry's options that allows
+                /// further customization.
+                #[inline]
+                pub unsafe fn set_handler_fn_unchecked(
```

Comment:
This method should also have a different name that indicates that an additional `cs_segment` is expected as argument.